Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -1,17 +1,649 @@
 {
   "version": "Notebook/1.0",
   "items": [
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Utilization",
+            "subTarget": "1",
+            "postText": "",
+            "style": "secondary"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Anomalies",
+            "subTarget": "2",
+            "style": "secondary"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Host Config",
+            "subTarget": "3",
+            "style": "secondary"
+          }
+        ]
+      },
+      "conditionalVisibility": {
```

Comment:
the first step of this workbook appears to have a conditionalVisibility setting that makes it always visible?  might want to just delete that part?

Reply:
Actually, the condition testDataExist != (empty) checks if the ingested data has already "arrived" to Log Analytics. We use it pretty much in every step and display an error message if the data is not there yet.

Reply:
yes, but this is the tabs very first step of the workbook, which exists before that parameter exists?

Reply:
Good catch - fixed